.gitignore

```diff
@@ -34,6 +36,7 @@ kolibrihome_test/
 
 # Translations
 kolibri/locale/en/
+kolibri/locale/**/django.mo
```

Comment:
Yes I'm not pretty sure why these files were generated so I git-ignored them. Do you have any suggestions where I should put this line? Or do you think I can just add it to the repository? Thank you!